src/components/NewCommentForm.tsx

```diff
@@ -41,28 +132,35 @@
 
         <div className="control has-icons-left has-icons-right">
           <input
+            value={authorEmail}
+            onChange={handleAuthorEmailChange}
             type="text"
```

Comment:
The input type for the email field should be `email` instead of `text` to ensure proper validation and user experience. Change `type="text"` to `type="email"`.